packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -57,10 +68,11 @@ const style: ComponentStyles<ConfigurationPanelManagedClasses, DevSiteDesignSyst
             padding: `${toPx(6)} ${toPx(8)}`,
             margin: toPx(2),
             minWidth: toPx(25),
+            color: "white",
             backgroundPosition: "center",
             "&:hover": {
                 cursor: "pointer",
-                background: "rgba(0, 0, 0, 0.04)",
+                background: "rgba(255, 255, 255, 0.04)",
```

Comment:
Assuming this doesn't connect to the design system provider

Reply:
We should do a pass on these later when we hook up the themes on the dev site

Reply:
@eljefe223 I'm curious if we should target a feature branch then so we can push the feature as "complete". I wouldn't be comfortable with the possibility of this deploying without a user being able to opt-out of dark theme and opt-in to light (or their own). Let's discuss further.

Reply:
@chrisdholt sounds like a good plan to me. @janechu @nicholasrice thoughts?

Reply:
The review title is "add dark theme and set as default" - can we *not* set it as default?

Reply:
This will set it as "the way it is" with no way to change